packages/react-core/src/Authenticator/hooks/useAuthenticator/__tests__/__snapshots__/useAuthenticator.spec.tsx.snap

```diff
@@ -7,7 +7,7 @@ exports[`useAuthenticator returns the expected values 1`] = `
     "EMAIL",
     "TOTP",
   ],
-  "authStatus": "authenticated",
+  "authStatus": "configuring",
```

Comment:
Updated due to difference in behavior on removal of `waitForNextUpdate`, does not impact production code